### ISSUE TITLE
Tutorial 2 IMDb data split

### ISSUE THREAD

**Issue by ydxu16** [bentrevett/pytorch-sentiment-analysis#58]
Latest pytorch 1.3 torchtext 0.4 and python 3.7.5

Using 

train, test = datasets.IMDB.splits(text, label)

Then
Len(train) = 0
Len(test) = 21020

**Comment by bentrevett** [comment]
I have re-run notebook 2 just now and have not got the same error.

How are you defining your `text` and `label`?

**Comment by ydxu16** [comment]
> @ydxu16 I have re-run notebook 2 just now and load the dataset fine.
> 
> How are you defining your `text` and `label`?

Exactly same as you defined. Random seed is same too. The set up is in AWS

**Comment by ydxu16** [comment]
> @ydxu16 I have re-run notebook 2 just now and load the dataset fine.
> 
> How are you defining your `text` and `label`?
I just re-ran your code by copy paste.

**Comment by ydxu16** [comment]
> @ydxu16 I have re-run notebook 2 just now and load the dataset fine.
> 
> How are you defining your `text` and `label`?

One diff might be this is run on CPU not GPU

**Comment by bentrevett** [comment]
I don't think the CPU/GPU is an issue.

Is there a chance the IMDB data became corrupted during the download? Check the `.data/imdb/aclImdb/train/pos` and `.data/imdb/aclImdb/train/neg` both contain 12500 .txt files and check a few to see if they are not empty.

**Comment by ydxu16** [comment]
> I don't think the CPU/GPU is an issue.
> 
> Is there a chance the IMDB data became corrupted during the download? Check the `.data/imdb/aclImdb/train/pos` and `.data/imdb/aclImdb/train/neg` both contain 12500 .txt files and check a few to see if they are not empty.

Thanks for your message. I will try again and will let you know once I get some results.

**Comment by bentrevett** [comment]
@ydxu16 Let me know if your issue has been solved and I can close this.

**Comment by ydxu16** [comment]
> @ydxu16 Let me know if your issue has been solved and I can close this.

Yes that solves the problem. Thanks!
